lib/CONST.jsx

```diff
@@ -348,6 +348,10 @@ export const CONST = {
             DEV: 'https://s3.amazonaws.com/dev-receipts-expensify-com/',
             PROD: 'https://s3.amazonaws.com/receipts.expensify.com/'
         },
+        RECEIPTS: {
+            DEV: 'https://www.expensify.com.dev/receipts/',
+            PROD: 'https://www.expensify.com/receipts/',
```

Comment:
I proposed something else (better?) but IDK if it's possible: https://expensify.slack.com/archives/C03TQ48KC/p1641495058018300?thread_ts=1641494910.018000&cid=C03TQ48KC

Reply:
I'm not sure how the proposal you linked is different from what you are doing here...

Reply:
I want it to make it dynamic dependent on an environment variable telling us whether it's staging or prod, but thinking more about this, I also thought that the calling code should do that and it's fine (like in my associated PR)